bin/ember-template-lint.js

```diff
@@ -373,9 +378,7 @@ async function run() {
     let hasErrors = results.errorCount > 0;
     let hasWarnings = results.warningCount > 0;
     let hasTodos = options.includeTodo && results.todoCount;
-    let hasUpdatedTodos =
-      options.updateTodo &&
-      (Number.isInteger(todoInfo.added) || Number.isInteger(todoInfo.removed));
+    let hasUpdatedTodos = options.updateTodo && todosToAdd;
```

Comment:
```js
    let hasUpdatedTodos = options.updateTodo && (todoInfo.added > 0 || todoInfo.removed > 0);
```

Like that ^

Reply:
Again, read above. We want to allow output whether you have created/removed or not, hence the boolean.

Reply:
OK, but then we should just do:

```suggestion
    let hasUpdatedTodos = options.updateTodo;
```

Because with the boolean (as far as I can tell) it is impossible for the `todosToAdd` to be a different value than `options.updateTodo`...

Reply:
Agree. Much easier.